middleware/queue/include/queue/common/ipc/message.h

```diff
@@ -795,6 +796,59 @@ namespace casual
                };
                
             } // remove
+            namespace recovery
+            {
+               enum class Type : int
+               {
+                  commit,
+                  rollback
+               };
```

Comment:
Is it not more like _Phase_? It's what we want the transaction state to be when we're done, right? So more like _Phase_ or _Directive_?